src/Aspire.Hosting.Azure.EventHubs/EventHubsEmulatorContainerImageTags.cs

```diff
@@ -5,12 +5,12 @@ namespace Aspire.Hosting.Azure.EventHubs;
 
 internal static class EventHubsEmulatorContainerImageTags
 {
-    /// <summary>mcr.microsoft.com</summary>
+    /// <remarks>mcr.microsoft.com</remarks>
```

Comment:
Don't we need to make this change everywhere?

Reply:
This change was already made everywhere in the release/9.0 branch. It just didn't make its way to main. We need to sync the code bases in a different PR (David Pine's one?).